### PR TITLE
Krishguptadev/feature-add-testimonials-160

### LINKED ISSUE #160: [FEATURE] Add Testimonials in creator page
### Detailed description

Hey folks! Another green square 🎉 

Let's add testimonials to the [creators](https://devprotocol.xyz/creators) page

### Context

This is very important to know the creators experience within the Dev Protocol

### Possible implementation

I suggest to use this
https://tailwindui.com/components/marketing/sections/testimonials#component-54ee9cbce13afa3082f12b00ff1c00a6
image

### Additional information

Here's their testimonials:

### Sindre 
"I like how the Dev Protocol is improving open source sustainability in an innovative way"

link: https://twitter.com/sindresorhus

### μ-zomia
"DevProtocol is born out of the love of people supporting people. Once you understand that, there's no reason not to be part of the team."

link: https://twitter.com/zomia12

### HiDE
"DEV protocol is a sustainable source of funding for the community ecosystem that uses our services to operate.
It is a core feature of the HiÐΞ protocol and is a very important financial resource. As OSS creators, we feel free to use some DEV tokens for development and redistribute them in our services. I will continue to do my best for the further development of the DEV community that you have staked out for us."

link: https://twitter.com/HideProtocol

### ScytheSwap 
"Being a part of Dev Protocol has been an incredible experience. When I was on boarded to stakes.social I was exposed to an audience of OSS creators and enthusiasts alike who helped to fund our project while simultaneously engaging with us, creating long-term connections with some of the most intelligent developers in the world. Dev Protocol, and all that it offers to OSS contributors, along with its community are truly one of a kind."

link: https://github.com/ScytheSwapProtocol

### IslandviewDigital
"Being part of Dev Protocol not only means being able to sustainably raise capital for our immersive, revenue-generating VR real estate development projects, it means being part of a community of talented innovators with whom we can synergize to bring even more value to the protocol."

link: https://twitter.com/IslandVDigital

### Samuil / SQ-UI
"DEV protocol jump-started our project and helped us do what we love."

link: https://twitter.com/samuil4

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by krishguptadev**
fixes #160

<a href="https://gitpod.io/#https://github.com/web3community/devprotocol.xyz/pull/161"></a>

### CODE REVIEW

**Comment by krishguptadev** [comment]
@vinzvinci I need images...

**Comment by vinzvinci** [comment]
> @vinzvinci I need images...

I'll try to gather them, please wait.

**Review by vinzvinci — CHANGES_REQUESTED** [review]
Can u put a title? 

title: Testimonials

**Comment by krishguptadev** [comment]
> Can u put a title?
> 
> title: Testimonials

I don't think testimonials usually have titles.

**Comment by vinzvinci** [comment]
> > Can u put a title?
> > title: Testimonials
> 
> I don't think testimonials usually have titles.

I think it is better to put **Testimonials** than none, it's not good at jumping to its content.

File: src/components/Creators/Testimonials.vue [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+          name: "μ-zomia",
+          image: "/assets/m-zomia.png",
+          description: `"DevProtocol is born out of the love of people supporting people. Once you understand that, there's no reason not to be part of the team."`,
+          link: "https://twitter.com/zomia12",
+        },
+        {
+          name: "HiDE",
+          image: "/assets/hide.png",
+          description: `"DEV protocol is a sustainable source of funding for the community ecosystem that uses our services to operate.
+It is a core feature of the HiÐΞ protocol and is a very important financial resource. As OSS creators, we feel free to use some DEV tokens for development and redistribute them in our services. I will continue to do my best for the further development of the DEV community that you have staked out for us."`,
```
Comment (Panquesito7) [inline src/components/Creators/Testimonials.vue:240]:
```suggestion
It is a core feature of the HiÐΞ protocol and is a very important financial resource. As OSS creators, we feel free to use some DEV tokens for development and redistribute them to our services. I will continue to do my best for the further development of the DEV community that you have staked out for us."`,
```

Comment (vinzvinci) [reply src/components/Creators/Testimonials.vue:240]: I will contact HiDE about this, this is a testimonial, we can't change anything unless they allow it.

Comment (krishguptadev) [reply src/components/Creators/Testimonials.vue:240]: We can't change someone's testimonial, right? 🙂

Comment (vinzvinci) [reply src/components/Creators/Testimonials.vue:240]: Yes.

File: src/components/Creators/Testimonials.vue [hunk 2]
```diff
@@ -0,0 +1,312 @@
+<template>
+  <section class="bg-white dark:bg-darkPrimary overflow-hidden text-center px-8">
+    <h2 class="mx-auto w-full font-display text-5xl m-8 dark:text-white">Community Partners</h2>
```
Comment (vinzvinci) [inline src/components/Creators/Testimonials.vue:3]: ~~Community Partners~~ Testimonials

Comment (Panquesito7) [reply src/components/Creators/Testimonials.vue:3]: Good catch, @vinzvinci. 👀
```suggestion
    <h2 class="mx-auto w-full font-display text-5xl m-8 dark:text-white">Testimonials</h2>
```

Comment (krishguptadev) [reply src/components/Creators/Testimonials.vue]: This is what happens when you have two gitpod tabs open together 🤦

**Review by vinzvinci — CHANGES_REQUESTED** [review]
Hmm sorry, one last request, the current m-zomia logo still doesn't look good for circular.

can u try this https://twitter.com/zomia12/photo?

**Review by vinzvinci — COMMENTED** [review]
can someone check if m-zomia is showing 
image

File: src/components/Creators/Testimonials.vue [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+      items: [
+        {
+          name: "Sindre",
+          image: "https://github.com/sindresorhus.png",
+          description: `"I like how the Dev Protocol is improving open source sustainability in an innovative way"`,
+          link: "https://twitter.com/sindresorhus",
+        },
+        {
+          name: "μ-zomia",
+          image: "/assets/m-zomia-rounded.png",
```
Comment (krishguptadev) [inline src/components/Creators/Testimonials.vue:233]:
```suggestion
          image: "/assets/m-zomia-rounded.jpg",
```

**Review by vinzvinci — APPROVED** [review]
Now looks good to me 🎉

**Review by Panquesito7 — COMMENTED** [review]
Hmm, not sure if this looks good (especially in dark mode). 😅

image

**Comment by krishguptadev** [comment]
> Hmm, not sure if this looks good (especially in dark mode). 😅
> 
> image

I added the code from tailwindui 😅 Cc @vinzvinci

**Comment by vinzvinci** [comment]
> > Hmm, not sure if this looks good (especially in dark mode). 😅
> > image
> 
> I added the code from tailwindui, Also it looks fine IMHO. 😅 Cc @vinzvinci

It still looks fine for me. We are required to use TailwindUI xD. I think soon we will follow the design system of what matteu build (hashi)

**Review by KukretiShubham — APPROVED** [review]
LGTM

**Review by Panquesito7 — COMMENTED** [review]
Nice! 🚀

**Pull request merged**